code/obj/item/clothing/hats.dm

```diff
@@ -10,6 +10,7 @@
 	compatible_species = list("human", "cow", "werewolf", "flubber")
 	var/seal_hair = 0 // best variable name I could come up with, if 1 it forms a seal with a suit so no hair can stick out
 	block_vision = 0
+	var/path_prot = 1 // protection from airborne pathogens, multiplier for chance to be infected
```

Comment:
TRUE/FALSE

Reply:
It's intended to be a continuous multiplier, I just elected to make all the stuff that I thought of completely protect right now, but there may be stuff that only lowers infection risk in the future.